internal/repository/aircraft.go

```diff
@@ -0,0 +1,77 @@
+package repository
+
+import (
+	"github.com/avialog/backend/internal/model"
+	"gorm.io/gorm"
+)
+
+type AircraftRepository interface {
+	Save(aircraft model.Aircraft) (model.Aircraft, error)
+	GetById(id uint) (model.Aircraft, error)
```

Comment:
Change `..ById` here and in other places to `..ByID` as your naming convention does not follow Golang idiomatic codestyle guidelines: https://stackoverflow.com/a/38766632/6666990

Reply:
Yes sir! I was just checking to see if you're reviewing the code carefully.